### PR TITLE
NFC: Standardizing file line endings

### PR DESCRIPTION
**Pull request by jsiirola**
## Summary/Motivation:
This fixes line endings throughout the repo (replacing CRLF with CR)

## Changes proposed in this PR:
- `dos2unix` on the `.py`, `.txt`, `.rst`, `.c`, `.h`, `.cpp`, `.hpp` files in the repo

### Legal Acknowledgement

By contributing to this software project, I have read the [contribution guide](https://pyomo.readthedocs.io/en/stable/contribution_guide.html) and agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the BSD license.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: doc/OnlineDocs/contributed_packages/satsolver.rst [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+    pip install z3-solver
+
+Using z3 Sat Solver
+-------------------
+To use the sat solver define your pyomo model as usual:
+
+.. doctest::
+
+  Required import
+  >>> from pyomo.environ import *
```
Comment (mrmundt) [inline doc/OnlineDocs/contributed_packages/satsolver.rst:22]: We don't care if the users use `import *`, correct?

**Review by mrmundt — APPROVED** [review]

File: doc/OnlineDocs/contributed_packages/satsolver.rst (hunk 1 above)
Comment (jsiirola) [reply doc/OnlineDocs/contributed_packages/satsolver.rst:22]: It's not really recommended any more and we are trying to standardizing the documentation around using `import pyomo.environ as pyo`.  However, that change is unrelated to this PR.

Comment (mrmundt) [reply doc/OnlineDocs/contributed_packages/satsolver.rst:22]: Fair enough!

**Comment by mrmundt** [comment]
@jsiirola  - I'm going to merge this, but if you want to go down the codecov/patch rabbit hole in a different PR, be my guest.

**Pull request merged**